core/Flist.cv64a6_imacfd_sv39

```diff
@@ -0,0 +1,153 @@
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
Super nitpick: The isa string is `imafdc` not `imacfd`

Reply:
Yes, super nitpick!!!! :-)     Fixed by commit